tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -1264,14 +1264,14 @@ struct LoadEnumerationsResponse {
 struct LoadArraySchemaRequest {
   config @0 :Config;
   # Config
-
-  includeEnumerations @1 :Bool;
```

Comment:
Unfortunately I think you might risk crashing older clients if you do this. Please check https://github.com/TileDB-Inc/TileDB/blob/f84be6c6da6f417754873638745d9bd7bf153a88/tiledb/sm/serialization/array_schema.cc#L1909
which is read checking if exists on the message :(

Reply:
Yeah bummer, it was exposed in #4399. I added it back and used it for this option, I documented that future options like this should be stored in the serialized Config.

Reply:
As a general note, whenever we see a field accessed without an `if (has)` wrapped around it, we should at the minimum ask a question about it, these things bite hard and lock us into supporting fields forever.

Reply:
Thanks! I added the `hasConfig` check during deserialization. I reran tests this morning in #5181 with these changes and everything is passing, this is ready for review when you have some time.